src/components/Navbar.vue

```diff
@@ -38,33 +37,41 @@
             Publications
           </b-dropdown-item>
 
-          <b-dropdown-item
-            to="/about/citing">
-            Citing Monarch
+          <b-dropdown-item href="https://medium.com/@MonarchInit" target="_blank" rel="noopener noreferrer">
+            Blog
           </b-dropdown-item>
 
-          <b-dropdown-item
-            href="https://medium.com/@MonarchInit"
-            target="_blank"
-            rel="noopener noreferrer">
-            Blog
+          <b-dropdown-item href="https://github.com/monarch-initiative" target="_blank" rel="noopener noreferrer">
+            GitHub Organization
           </b-dropdown-item>
 
-          <b-dropdown-item to="/sources">
-            Sources
+          <b-dropdown-divider/>
+
+          <b-dropdown-item href="https://archive.monarchinitiative.org/latest/" target="_blank" rel="noopener noreferrer">
+            Data Downloads
           </b-dropdown-item>
 
-          <b-dropdown-item
-            href="https://github.com/monarch-initiative"
-            target="_blank"
-            rel="noopener noreferrer">
-            GitHub Organization
+          <b-dropdown-item to="/sources">
+            Data Sources
           </b-dropdown-item>
+
         </b-nav-item-dropdown>
+        <b-nav-item-dropdown text="Help">
+          <b-dropdown-item to="/help/contact">
+            Contact Us
+          </b-dropdown-item>
+
+          <b-dropdown-item to="/help/citing">
+            Citing Monarch
+          </b-dropdown-item>
+
+          <b-dropdown-item to="/help/links">
+            Link to Us
+          </b-dropdown-item>
+          <b-dropdown-divider/>
 
-        <b-nav-item-dropdown text="Examples">
           <b-dropdown-header class="text-info bg-white">
-            Interesting nodes for Demo or Debugging
+            Examples
```

Comment:
I would like to remove the the lengthy list of examples from the 'Help' menu. 

@iimpulse you were sharing that vue does not have a way to implement sub-menus under the items listed in the drop-down menus. If we can find a way to implement a submenu under the item 'Examples,' then let's keep it under 'Help.' Else, I suggest that we make a new top-level menu with the word 'Example searches' and take that lengthy list off the 'Help' menu, please.
@pnrobinson would that be okay with you too?

Reply:
These changes have been reverted.

Reply:
it's more that bootstrap doesn't support them, https://github.com/twbs/bootstrap/pull/6342#issuecomment-11594010 but I'm guessing there are hacky ways around it.  Also see https://github.com/bootstrap-vue/bootstrap-vue/issues/3435

Reply:
Yeah.. lets just have it how it is, and if bootstrap vue goes to implement, I'll follow suite. Although when we go to prod I think we should stick to the tags under the search bar

Reply:
Thank you!